packages/expo-errors/ios/EXErrors/EXErrors.h

```diff
@@ -0,0 +1,11 @@
+// Copyright 2018-present 650 Industries. All rights reserved.
+
+#import <Foundation/Foundation.h>
+
+#import <EXCore/EXDefines.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+void EXRejectInvalidArgument(NSString *message, EXPromiseRejectBlock rejecter);
```

Comment:
What do you think about swapping these args so that if we wanted to add a printf-like formatting API it'd be easier to tack the varargs on at the end?